lib/glific/providers/gupshup/partner/partner_api.ex

```diff
@@ -92,6 +92,23 @@ defmodule Glific.Providers.Gupshup.PartnerAPI do
     end
   end
 
+  @doc """
+  Enable template messaging for an app.
+  """
+  @spec enable_template_messaging(non_neg_integer()) :: {:ok, map()} | {:error, any}
+  def enable_template_messaging(org_id) do
+    url = app_url(org_id) <> "/appPreference"
+    data = %{"isHSMEnabled" => "true"}
+
+    case put_request(url, data, org_id: org_id) do
+      {:ok, response} ->
+        {:ok, response}
+
+      {:error, error} ->
+        {:error, "Error during enable template messaging: #{inspect(error)}"}
```

Comment:
Can you update this to return response received from Gupshup.
<img width="1440" alt="image" src="https://github.com/glific/glific/assets/40158831/cb0f8abc-3108-4341-bb0e-d2f1eeeaeb02">